src/styles/utils/variables.scss

```diff
@@ -0,0 +1,19 @@
+$height-box: 100px;
+$width-box: 100px;
+$color: #000;
+$transition-time: 0.5s;
+$padding-page: 10px;
+$width-page: 7 * $width-box + 21px;
+$week-day:
+  "mon" 0,
+  "tue" $width-box + 1px,
+  "wed" 2*$width-box + 2*1px,
+  "thu" 3*$width-box + 3*1px,
+  "fri" 4*$width-box + 4*1px,
+  "sat" 5*$width-box + 5*1px,
+  "sun" 6*$width-box + 6*1px;
```

Comment:
This is okay and working, but you can achieve this by only defining day and number here and all the calculation in your start day class. Just a suggestion. apart from this, while doing any calculation you can do code formatting like proper spacing and indentation